apps/marketplace/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,29 @@
+﻿import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+const routes: Routes = [
+  {
+    path: '',
+    redirectTo: 'landing',
+    pathMatch: 'full'
+  },
+  {
+    path: 'landing',
+    loadChildren: () => import('./landing/landing.module').then(m => m.LandingModule) 
```

Comment:
can't find the LandingModule file in your PR?

Reply:
good catch!.. thanks :)

Reply:
git added the files.